### DISCUSSION TITLE
Google Drive link not accepting, using v0.73.1 of outline

### DISCUSSION THREAD

**Discussion by hareenpu-kore** [outline/outline#6260] [General]
On pages, i have tried to add the Google Drive link (https://drive.google.com/drive/folders/XXXXXXXXXXXXXXXXXXXXX?usp=sharing)

<img width="513" alt="image" src="https://github.com/outline/outline/assets/136708702/996dda93-d975-4bed-a459-52865924fa7e">

**Comment by tommoor** [comment]
Hi, it's only possible to embed files – not folders, this is a restriction on Google's side.

**Reply by hareenpu-kore** [reply]
We are trying to embed a file, where the file name changes daily, so we provided the complete path but did not take the link .

if we provide link with the file , we can access but by the next day file name changes so we are getting no file found
